front/src/applications/operationalStudies/views/SimulationResults.tsx

```diff
@@ -260,7 +267,7 @@ const SimulationResults = ({
         </div>
       </ResizableSection>
 
-      {simulationResults && (
+      {simulationResults && simulationResults.isValid && (
```

Comment:
It should be fine like this :

```suggestion
      {simulationResults?.isValid && (
```

Reply:
Done!